### DISCUSSION TITLE
Backoffice dataschema: how to flat an object?

### DISCUSSION THREAD

**Discussion by edocrippaofficial** [mia-platform/community#181] [Q&A] [answered]
Hi, I have a Mongo object with the following properties:
```
{
  field1,
  field2,
  metadata: {
    field3  
  }
}
```

Is there a way to display it in a bk-table with the following columns?
| field1 | field2 | field3 |

**Comment by giuliowaitforitdavide** [comment, marked as answer]
You could use a MongoDB View to map your field3 in the first level of your object (like field1 and field2) and then display the view itself

**Reply by edocrippaofficial** [reply]
Yes I thought about that but I was wondering if it was natively supported by back-kit, running an aggregation pipieline every time a user request a page is maybe too much for my use case

**Reply by giuliowaitforitdavide** [reply]
No unfortunately there isn't 😢

**Reply by clabene** [reply]
@giuliowaitforitdavide is correct, back-kit components do not support such feature atm

**Reply by edocrippaofficial** [reply]
Ok, thanks anyways :+1:
